pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -723,6 +723,10 @@ type VerticalPodAutoscaler struct {
 	// (default: 0.95)
 	// +optional
 	RecommendationUpperBoundCPUPercentile *float64 `json:"recommendationUpperBoundCPUPercentile,omitempty" protobuf:"fixed64,11,opt,name=recommendationUpperBoundCPUPercentile"`
+	// CPUHistogramDecayHalfLife is the amount of time it takes a historical CPU usage sample to lose half of its weight.
+	// (default: 24h)
+	// +optional
+	CPUHistogramDecayHalfLife *metav1.Duration `json:"cpuHistogramDecayHalfLife,omitempty" protobuf:"bytes,15,opt,name=cpuHistogramDecayHalfLife"`
```

Comment:
Can you move this down as well so that the protobuf ordering is properly? It's somewhat confusing atm

Reply:
The properties are currently ordered by semantics rather than protobuf id. Settings for memory histogram and settings for cpu histogram are in their own groups. This made more sense to me than grouping by something arbitrary like the protobuf id – which essentially just means the order in which we added these to our API.

I guess if we're changing it here, we should have the same order as well in `pkg/component/autoscaling/vpa/recommender.go`, `pkg/component/autoscaling/vpa/vpa_test.go` and `pkg/gardenlet/operation/botanist/vpa.go` for the parameters.

If you insist, I will change the order – I don't think this improves the situation for a human looking at this, though.

Reply:
In the past, we had cases where people tried the same and accidentally changed the protobuf numbers (which is a breaking API change). Hence, we usually recommend to just put new fields to the end of the list (even if they fit better somewhere else semantically). Whether you change it in the VPA component API doesn't matter much (this is no external-facing API). My personal preference for those is alphabetical ordering, but feel free to do whatever you like.

Reply:
done.